webpack/loaders.js

```diff
@@ -1,14 +1,5 @@
 'use strict';
 
-exports.angular = { // ships in ES6 format now
-  test: /\.js$/,
-  loader: 'babel-loader',
-  include: /angular/,
-  exclude: /node_modules/,
-  query: {
-    compact: false,
-  },
-};
```

Comment:
Was this interfering with AOT? If so, we should remove the package.json dependencies. Beware that `awesome-typescript-loader` is using Babel to go from ES6 to ES5 and may have some cross dependencies.